modules/rosa-quick-start-install-configure-cli-tools.adoc

```diff
@@ -0,0 +1,159 @@
+// Module included in the following assemblies:
+//
+// * rosa_quick_start/rosa-quick-start.adoc
+
+[id="rosa-quick-start-install-configure-cli-tools_{context}"]
+= Installing and configuring the required CLI tools
+
+Use the following steps to install and configure the AWS, {product-title} (ROSA) and OpenShift CLI tools on your workstation.
+
+.Prerequisites
+
+* You created an AWS account.
```

Comment:
can we instead state that they "have" an aws and red hat account?  Small change but they dont need to create a new one.

Reply:
Thanks for the suggestion. I have updated this throughout the PR, except in the first instance where we provide more detail on using your existing org or creating a new one.